jbpm-runtime-manager/src/test/java/org/jbpm/runtime/manager/impl/deploy/DeploymentDescriptorMergerTest.java

```diff
@@ -41,498 +41,6 @@
-		assertEquals("org.jbpm.domain", master.getAuditPersistenceUnit());
-		assertEquals(AuditMode.JPA, master.getAuditMode());
-		assertEquals(PersistenceMode.JPA, master.getPersistenceMode());
-		assertEquals(RuntimeStrategy.SINGLETON, master.getRuntimeStrategy());
-		assertEquals(0, master.getMarshallingStrategies().size());
-		assertEquals(0, master.getConfiguration().size());
-		assertEquals(0, master.getEnvironmentEntries().size());
-		assertEquals(0, master.getEventListeners().size());
-		assertEquals(0, master.getGlobals().size());
-		assertEquals(0, master.getTaskEventListeners().size());
-		assertEquals(1, master.getWorkItemHandlers().size());
-
-		DeploymentDescriptor slave = new DeploymentDescriptorImpl("org.jbpm.domain");
-
-		slave.getBuilder()
-		.auditMode(AuditMode.JMS)
-		.persistenceMode(PersistenceMode.JPA)
-		.persistenceUnit(null)
-		.auditPersistenceUnit("")
-		.addWorkItemHandler(new NamedObjectModel("mvel", "Log", "new org.jbpm.process.instance.impl.demo.CustomSystemOutWorkItemHandler()"));
-
-		assertNotNull(slave);
-		assertEquals(null, slave.getPersistenceUnit());
-		assertEquals("", slave.getAuditPersistenceUnit());
-		assertEquals(AuditMode.JMS, slave.getAuditMode());
-		assertEquals(PersistenceMode.JPA, slave.getPersistenceMode());
-		assertEquals(RuntimeStrategy.SINGLETON, slave.getRuntimeStrategy());
-		assertEquals(0, slave.getMarshallingStrategies().size());
-		assertEquals(0, slave.getConfiguration().size());
-		assertEquals(0, slave.getEnvironmentEntries().size());
-		assertEquals(0, slave.getEventListeners().size());
-		assertEquals(0, slave.getGlobals().size());
-		assertEquals(0, slave.getTaskEventListeners().size());
-		assertEquals(1, slave.getWorkItemHandlers().size());
-
-		// and now let's merge them
-		DeploymentDescriptorMerger merger = new DeploymentDescriptorMerger();
-		DeploymentDescriptor outcome = merger.merge(master, slave, MergeMode.MERGE_COLLECTIONS);
-
-		assertNotNull(outcome);
-		assertEquals("org.jbpm.domain", outcome.getPersistenceUnit());
-		assertEquals("org.jbpm.domain", outcome.getAuditPersistenceUnit());
-		assertEquals(AuditMode.JMS, outcome.getAuditMode());
-		assertEquals(PersistenceMode.JPA, outcome.getPersistenceMode());
-		assertEquals(RuntimeStrategy.SINGLETON, outcome.getRuntimeStrategy());
-		assertEquals(0, outcome.getMarshallingStrategies().size());
-		assertEquals(0, outcome.getConfiguration().size());
-		assertEquals(0, outcome.getEnvironmentEntries().size());
-		assertEquals(0, outcome.getEventListeners().size());
-		assertEquals(0, outcome.getGlobals().size());
-		assertEquals(0, outcome.getTaskEventListeners().size());
-		assertEquals(1, outcome.getWorkItemHandlers().size());
-
-		// let's check if the slave version is preserved
-		NamedObjectModel model = outcome.getWorkItemHandlers().get(0);
-		assertEquals("Log", model.getName());
-		assertEquals("new org.jbpm.process.instance.impl.demo.CustomSystemOutWorkItemHandler()", model.getIdentifier());
-	}
-
```

Comment:
Have all these tests been deleted because they had been moved previously to droolsjbpm-knowledge?

Reply:
yes, I changed my mind about having them duplicated.

Reply:
Following @elguardian advice, rather than removing the test, just changing master and slave